Assets/_Scripts/FightingEntity.cs

```diff
@@ -16,6 +16,13 @@ public class FightingEntity : MonoBehaviour
 	public List<ActionBase> actions = new List<ActionBase>();
 
     public int targetId;
+    public string targetName;
+
+    public FighterSlot slot {get; set;}
```

Comment:
The FighterSlot should have a reference of the FightingEntity, not the other way around.

Reply:
It can be both ways if we need it to be, but currently I only need to player to have a reference to the FighterSlot to figure out if X is flipped. Changing it to only be the other way around would require an unnecessary amount of work to our current infrastructure (Every instance where we use player we need to use PlayerSlot instead)

Reply:
Made the change